rakelib/lint.rake

```diff
@@ -0,0 +1,22 @@
+namespace :lint do
+  desc 'Install swiftlint'
+  task :install do |task|
+    swiftlint = `which swiftlint`
```

Comment:
After googling it seems `system('which swiftlint')` is the easiest way and directly returns true if successful, false otherwise